afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -27,9 +37,15 @@
 
     private final WeakListenerList<AppStorageListener> listeners;
 
-    public NodeEventClient(String fileSystemName, WeakListenerList<AppStorageListener> listeners) {
+    private final RemoteServiceConfig config;
+
+    private int reconnectionInterval;
```

Comment:
I would rename to currentReconnectionInterval in order to remove the ambiguity.

Reply:
It's done